syntax = "proto2";

package person;

message Person
{
    required string last_name = 1;
    required string first_name = 2;
    optional string middle_name = 3;
}
